### ISSUE TITLE
Add exponential backoff option to consumer.sh script

### ISSUE THREAD

**Issue by shouze** [DevOoops/docker#54]
At the moment the run loop don't take into account the exit status of the executed sub process:
https://github.com/DevOoops/docker/blob/2043501bbdae1e8531142d65cb518ec0ba1d4bea/php/consumer.sh#L35-L39

- [ ] Apply exponential back off principles in case of non `zero` exit code: sleep 2^ATTEMPT_NUMBER, capped to max sleep time (default, 2, can be set via `-s` option)

**Comment by shouze** [comment]
You will have to collect the exit code of the sub process this way:
```shell
    "$@" &
    SUBPROCESS_ID=$!
    wait $SUBPROCESS_ID
    exitCode=$?
```

You can take some inspiration from [this post](https://coderwall.com/p/--eiqg/exponential-backoff-in-bash)

**Comment by shouze** [comment]
Closed by #55
